### PR TITLE
Add parameters to the script

### PR DESCRIPTION
**Pull request by pranaysahith**

### CODE REVIEW

File: .github/workflows/run_tests.yaml [hunk 1]
```diff
@@ -30,10 +30,10 @@ jobs:
...
           pip install boto3
           cd jmeter-icap-poc/cloudformation
-          python3 create_stack.py --total_users ${TOTAL_USERS:-20} --ramp_up=${RAMP_UP:-20} --duration=${DURATION:-900} --influx_host=${INFLUX_HOST:-10.112.0.112} --endpoint_url=${ENDPOINT_URL:-gw-icap-k8s-a0c293ac.hcp.uksouth.azmk8s.io} --prefix=${PREFIX:-ga-}
+          python3 create_stack.py --total_users ${TOTAL_USERS:-20} --ramp_up=${RAMP_UP:-20} --duration=${DURATION:-900} --influx_host=${INFLUX_HOST:-10.112.0.112} --endpoint_url=${ENDPOINT_URL:-gw-icap-k8s-a0c293ac.hcp.uksouth.azmk8s.io} --prefix=${PREFIX:-ga}
         env:
-          BUCKET: ${{ secrets.BUCKET }}
-          FILE_NAME: ${{ secrets.FILE_NAME }}
+          SCRIPT_BUCKET: ${{ secrets.SCRIPT_BUCKET }}
+          SCRIPT_NAME: ${{ secrets.SCRIPT_NAME }}
           INFLUX_HOST: ${{ secrets.INFLUX_HOST }}
```
Comment (jpoolGW) [inline .github/workflows/run_tests.yaml:37]: Add parameter here for data source file

File: jmeter-icap-poc/scripts/config.env.sample [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+USERS_PER_INSTANCE=4000
+INSTANCES_REQUIRED=1
+RAMP_UP=300
+DURATION=900
+ENDPOINT_URL=gw-icap-k8s-a0c293ac.hcp.uksouth.azmk8s.io
+INFLUX_HOST=10.112.0.112
+PREFIX=
+INSTANCES_REQUIRED=1
+JMX_SCRIPT_NAME=ICAP_Direct_FileProcessing_v1.jmx
+GRAFANA_URL=10.112.0.112:3000
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/config.env.sample:18]: Put place holder here as this is not correct value

**Review by jpoolGW — CHANGES_REQUESTED** [review]

File: .github/workflows/run_tests.yaml (hunk 1 above)
Comment (pranaysahith) [reply .github/workflows/run_tests.yaml]: Added TEST_DATA_FILE env variable

File: jmeter-icap-poc/scripts/config.env.sample (hunk 2 above)
Comment (pranaysahith) [reply jmeter-icap-poc/scripts/config.env.sample]: added a placeholder

**Review by jpoolGW — APPROVED** [review]

**Pull request merged**